clickhouse_backend/backend/schema.py

```diff
@@ -213,10 +213,14 @@ def _model_indexes_sql(self, model):
         )
         if (
             any(field.db_index for field in model._meta.local_fields)
-            or model._meta.index_together
+            or any(len(index.fields) > 1 for index in model._meta.indexes)
```

Comment:
Explicitly defined indexes are allowed, you don't need to check model._meta.indexes.

Change this to `or getattr(model._meta, "index_together", None)` and delete line 220-223

Reply:
done!